3-async-await/solution.js

```diff
@@ -22,14 +22,62 @@ Example:
 
 function solution() {
     // YOUR SOLUTION GOES HERE
+    const products = require('./products');
+    const prices = require('./prices');
 
     // You generate your id value here
+    const id = Number((Date.now()+'').slice(-2));
+    const result = {id};
 
-    // You use Promise.all() here
+    (async () => {
```

Comment:
convert solution function into async function instead of doing this

Reply:
ok, the challenge allowed to modify solution function.